crypto/evp/evp_local.h

```diff
@@ -65,6 +66,41 @@ struct evp_kdf_ctx_st {
     void *data;                 /* Algorithm-specific data */
 } /* EVP_KDF_CTX */ ;
 
+struct evp_rand_ctx_st {
+    EVP_RAND *meth;             /* Method structure */
+    void *data;                 /* Algorithm-specific data */
+    EVP_RAND_CTX *parent;       /* Parent seed source */
+    size_t max_request;         /* Cached: maximum number of bytes generated */
```

Comment:
is that max # of bytes that *can* be generated?

Reply:
Kind of, maximum number that can be generated in a single call.  I'll amend.